composer/datasets/streaming/upload.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import urllib.parse
+from typing import Optional
+
+from composer.utils.object_store.object_store import ObjectStore
+from composer.utils.object_store.s3_object_store import S3ObjectStore
+from composer.utils.object_store.sftp_object_store import SFTPObjectStore
+
+
+def get_object_store(remote: str, timeout: Optional[float]) -> ObjectStore:
```

Comment:
Nit: this can be a private helper
```suggestion
def _get_object_store(remote: str, timeout: Optional[float]) -> ObjectStore:
```